contrib/fuzz/archive_fuzzer.go

```diff
@@ -94,13 +95,14 @@ func FuzzImportIndex(data []byte) int {
 			hdrName := path.Clean(hdr.Name)
 			switch hdrName {
 			case ocispec.ImageLayoutFile:
-				break
+				hasLayoutOrManifest = true
 			case "manifest.json":
```

Comment:
nit: If we're already on the fixing old code kick 😆, could combine these two seeming as how they lead to the same outcome.

```go
switch hdrName {
case ocispec.ImageLayoutFile, "manifest.json":
    hasLayoutOrManifest = true
}
```

Reply:
Done!